packages/libs/error-stack/src/compat/mod.rs

```diff
@@ -0,0 +1,30 @@
+#[cfg(feature = "anyhow")]
+mod anyhow;
+#[cfg(feature = "eyre")]
+mod eyre;
+
+/// Compatibility trait to convert from external libraries to [`Report`].
+///
+/// *Note*: Is't not possible to implement [`IntoReport`] or [`Context`] on other error libraries
+/// types as both traits have blanked implementation relying on [`Error`]. This means, that they
+/// *could* implement [`Error`] on their own.
```

Comment:
could implement Error on their own which would cause problems because...

Reply:
Clarified the docs: b137334ba810097353b04a20127246f63378b6c3

Reply:
Perfect, that's **way** clearer. Thank you!